src/github/branch.ts

```diff
@@ -74,7 +74,7 @@ export const processBranch = async (
 		operationType: "NORMAL",
 		auditLogsource: "WEBHOOK",
 		entityAction: "BRANCH_CREATE",
-		subscriptionId: subscription?.id
+		subscriptionId: subscription?.id || 0
```

Comment:
This shouldn't happen, but we didn't `return` above if the `subscription` is empty, so just allow safe.